cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,64 @@
 /// <reference types='cypress' />
-
 describe('Book Store app', () => {
-  before(() => {
+  const testUser = {
+    username: 'wojtasck',
+    password: 'Wojtas1234!',
+    book: 'Speaking JavaScript',
+    description: 'Like it or not, JavaScript is everywhere these'
+  };
 
+  it('should login user with correct test data', () => {
+    cy.visit('/login');
```

Comment:
You can use your `login()` function here too.

Reply:
I wanted to do one manually for learning.